requirements/base.txt

```diff
@@ -9,6 +9,6 @@ kombu==4.6.1
 pytz==2019.1
 redis==3.2.1
 vine==1.3.0
-opentelemetry-distro==0.28b0
-opentelemetry-instrumentation==0.28b0
-opentelemetry-exporter-otlp==1.9.0
+opentelemetry-instrumentation==0.39b0
+opentelemetry-distro==0.39b0
+opentelemetry-exporter-otlp==1.18.0
```

Comment:
```suggestion
opentelemetry-exporter-otlp==1.18.0
importlib-metadata~=6.0.0
```

@zriyansh I am getting the error below when trying to install dependencies in python `3.10.3`:

```
ERROR: Exception:
Traceback (most recent call last):
  File "/Users/prashant/work/python-falcon-template/venv/lib/python3.10/site-packages/pip/_internal/cli/base_command.py", line 167, in exc_logging_wrapper
    status = run_func(*args)
  File "/Users/prashant/work/python-falcon-template/venv/lib/python3.10/site-packages/pip/_internal/cli/req_command.py", line 205, in wrapper
    return func(self, options, args)
  File "/Users/prashant/work/python-falcon-template/venv/lib/python3.10/site-packages/pip/_internal/commands/install.py", line 339, in run
    requirement_set = resolver.resolve(
  File "/Users/prashant/work/python-falcon-template/venv/lib/python3.10/site-packages/pip/_internal/resolution/resolvelib/resolver.py", line 94, in resolve
    result = self._result = resolver.resolve(
  File "/Users/prashant/work/python-falcon-template/venv/lib/python3.10/site-packages/pip/_vendor/resolvelib/resolvers.py", line 481, in resolve
    state = resolution.resolve(requirements, max_rounds=max_rounds)
  File "/Users/prashant/work/python-falcon-template/venv/lib/python3.10/site-packages/pip/_vendor/resolvelib/resolvers.py", line 373, in resolve
    failure_causes = self._attempt_to_pin_criterion(name)
  File "/Users/prashant/work/python-falcon-template/venv/lib/python3.10/site-packages/pip/_vendor/resolvelib/resolvers.py", line 227, in _attempt_to_pin_criterion
    raise InconsistentCandidate(candidate, criterion)
pip._vendor.resolvelib.resolvers.InconsistentCandidate: Provided candidate LinkCandidate('https://files.pythonhosted.org/packages/cc/37/db7ba97e676af155f5fcb1a35466f446eadc9104e25b83366e8088c9c926/importlib_metadata-6.8.0-py3-none-any.whl#sha256=3ebb78df84a805d7698245025b975d9d67053cd94c79245ba4b3eb694abe68bb (from https://pypi.org/simple/importlib-metadata/) (requires-python:>=3.8)') does not satisfy SpecifierRequirement('importlib-metadata~=6.0'), SpecifierRequirement('importlib-metadata~=6.0.0')
```

/cc @srikanthccv

Reply:
It may be because of `Cython==0.29.27` dependency, this version of cython does not support python 3+ as stated in below package manager. 

![Screenshot 2023-07-14 at 01 20 07](https://github.com/SigNoz/python-falcon-template/assets/52788043/776271bf-90f9-4ebf-bf1b-73f7f394dc5d)

@prashant-shahi 

https://pypi.org/project/Cython/0.29.21/